src/components/Profile/MyLaundryBookings.jsx

```diff
@@ -149,16 +149,17 @@ export default function MyLaundryBookings ({ loggedIn }) {
               <b>Mina tvättbokningar</b>{' '}
             </Card.Header>{' '}
             <br />
-            {laundryBookings.map((row) => (
-              <Card.Text className='border' key={row.start_time}>
-                <b>Starttid</b> : {row.start_time} <br /> <b>Sluttid</b> :{' '}
-                {row.end_time} <br />
+            {console.log(laundryBookings)}
```

Comment:
Is this supposed to still be here?

Reply:
```suggestion
```
I agree with @erikbengtssonchalmers this shouldn't be here.